README.md

```diff
@@ -3164,17 +3164,46 @@ tweak the implementation to your application's needs, you can follow the steps i
 In order to test Hazelcast C++ client locally, you will need the following:
 
 * Java 8 or newer (for server)
-* Maven
-* cmake
-* OpenSSL
-* Boost
-* thrift
+* Maven 3 or newer (for server)
+* cmake 3.10 or newer
+* OpenSSL 1.0.2 or newer
+* Boost >= 1.71
+* thrift 0.13
+* Bash (on platforms other than Windows)
+* Hazelcast Enterprise license
 
-You need to enable the examples in the build with the cmake flag `-DBUILD_EXAMPLES=ON`. When you build, the result will
+set max files limit ulimit -n
+
+You need to enable tests in the build with the cmake flag `-DBUILD_TESTS=ON`. When you build, the result will
 produce the executable `client_test` on Linux/macOS and `client_test.exe` on Windows. You need to start the remote
 controller first before running this executable by executing `scripts/start-rc.sh` on Linux/macOS
 or `scripts/start-rc.bat` on Windows. Then you can run the test executable.
 
+### 8.1.1 Testing on Linux/MacOS
+
+1. First, make sure you have all the requirements are met.
```

Comment:
```suggestion
1. First, make sure you have all the requirements satisfied.
```

Reply:
Thanks, I think I've mixed two different thoughts I had. I'll replace it with yours.